pages/linux/ss.md

```diff
@@ -10,9 +10,9 @@
 
 `ss {{state/exclude}} {{bucket/big/connected/synchronized/...}}`
 
-- Show all TCP sockets connected to the local HTTPS port (443):
+- Show all TCP sockets listening on the local HTTPS port (443):
```

Comment:
I'm somewhat confused here. This example shows the remote hosts connected to the local TCP port 443, and you've changed it to show the local process(es) listening on the local TCP port 443. Is that right?

There's still room on this page for another example - why remove this one?

Reply:
> you've changed it to show the local process(es) listening on the local TCP port 443. Is that right?

If by "process(es)" you means sockets, yes. I changed it since the page didn't have a 'listening' example. 

> There's still room on this page for another example - why remove this one?

I thought `ss -lt src :{{443}}` and `ss -pt dst :{{ssh}}` would be simple enough, would save some space and since the second one was already about "connected sockets" I thought maybe the first one could be about "listening" ones. To make the examples less repetitive and remain simple enough, imho. I don't mind changing it.

Reply:
Ah, I see. I'm still inclined to keep both examples though :-)